sechub-shared-kernel/src/main/java/com/daimler/sechub/sharedkernel/mapping/MappingIdentifier.java

```diff
@@ -12,16 +12,17 @@
 
 /**
  * Represents all mapping identifiers.
+ * 
  * @author Albert Tregnaghi
  *
  */
 @MustBeKeptStable("The enum is used as identifiers in database + in code. Do NOT rename Ids inside")
 public enum MappingIdentifier {
-   
-    CHECKMARX_NEWPROJECT_TEAM_ID("checkmarx.newproject.teamid", MappingType.ADAPTER_CONFIGURATION),
 
-    CHECKMARX_NEWPROJECT_PRESET_ID("checkmarx.newproject.presetid",MappingType.ADAPTER_CONFIGURATION),
-    
+    CHECKMARX_NEWPROJECT_TEAM_ID("checkmarx.newproject.teamid.mapping", MappingType.PRODUCT_EXEC_CFG_PARAM),
+
+    CHECKMARX_NEWPROJECT_PRESET_ID("checkmarx.newproject.presetid.mapping", MappingType.PRODUCT_EXEC_CFG_PARAM),
```

Comment:
Please change the name from `PRODUCT_EXEC_CFG_PARAM` to `PRODUCT_EXEC_CONFIG_PARAM`. For a detailed explanation why please look at my previous comment in line 22.

Reply:
I inspect our SQL etc. There is no need to reduce the names (mapping type is not used in persistence so does not matter).

So I will change as suggested to `PRODUCT_EXEC_CONFIG_PARAM`